tests/test_strf.py

```diff
@@ -1,8 +1,9 @@
-import matplotlib.pyplot as p
-p.switch_backend('Agg')
-from spykes.strf import STRF
 from nose.tools import assert_equal
 import numpy as np
+import matplotlib.pyplot as p
+p.switch_backend('Agg')
```

Comment:
I believe it just needs to be before the import for spykes. It is currently working, though, as  the Travis is building